WS2012R2/lisa/xml/DM_Tests.xml

```diff
@@ -181,21 +174,19 @@
 			<setupScript>SetupScripts\DM_CONFIGURE_MEMORY.ps1</setupScript>
 			<testParams>
 				<param>TC_COVERED=DM-07</param>
-				<param>vmName=VMName</param>
 				<param>enableDM=yes</param>
 				<param>minMem=1024MB</param>
 				<param>maxMem=60%</param>
 				<param>startupMem=60%</param>
 				<param>memWeight=0</param>
 				<param>staticMem=1024MB</param>
-				<param>vmName=VM2Name</param>
 				<param>enableDM=yes</param>
 				<param>minMem=1024MB</param>
 				<param>maxMem=40%</param>
 				<param>startupMem=40%</param>
 				<param>memWeight=0</param>
 				<param>staticMem=1024MB</param>
-				<param>vmName=VM3Name</param>
+				<param>vm3Name=VM3Name</param>
```

Comment:
I see that your changes are addressed in the VM section of the xml, however vm3 is still kept here.
Would it be the same to move vm3 in the xml VM section to keep the same new format proposed?
Thank you!